applications/ParticleMechanicsApplication/python_scripts/mpm_solver.py

```diff
@@ -236,7 +233,7 @@ def import_constitutive_laws(self):
             materials_imported = False
         return materials_imported
 
-    ### Solver private functions
+    ### Solver protected functions
 
     def _add_model_part_containers(self):
```

Comment:
are those protected or private?

=> check the [Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide#python-style-conventions) for naming of functions